libs/vkd3d/command.c

```diff
@@ -8064,7 +8053,7 @@ static void STDMETHODCALLTYPE d3d12_command_list_OMSetRenderTargets(d3d12_comman
     {
         graphics = &list->state->graphics;
 
-        if (prev_dsv_format != next_dsv_format && d3d12_graphics_pipeline_state_is_depth_stencil_sensitive(graphics))
+        if (prev_dsv_format != next_dsv_format && d3d12_graphics_pipeline_state_has_unknown_dsv_format(graphics))
```

Comment:
I think the code reads better if we also rename this to has_unknown_dsv_format_with_test_enabled or something like that to make it clear it's a workaround situation. Just "unknown_dsv_format" would normally mean it doesn't use depth-test at all, where "is depth stencil sensitive" is clearer for what it tries to tell.